docs/BreakingChanges/045 Error codes for maxRequestLength or maxReceivedMessageSize are different.md

```diff
@@ -0,0 +1,26 @@
+## 45: Error codes for maxRequestLength or maxReceivedMessageSize are different
```

Comment:
{
  "Id": "45",
  "Title": "Error codes for maxRequestLength or maxReceivedMessageSize are different",
  "Details": "Messages in WCF web services hosted in Internet Information Services (IIS) or ASP.NET Development Server that exceed maxRequestLength (in ASP.NET) or maxReceivedMessageSize (in WCF) have different error code\n\nThe HTTP status code has changed from 400 (Bad Request) to 413 (Request Entity Too Large), and messages that exceed either the maxRequestLength or the maxReceivedMessageSize setting throw a ProtocolException exception. This includes cases in which the transfer mode is Streamed.",
  "Suggestion": "This change facilitates debugging in cases where the message length exceeds the limits allowed by ASP.NET or WCF.\n\nYou must modify any code that performs processing based on an HTTP 400 status code.",
  "Link": "https://msdn.microsoft.com/en-us/library/hh367887(v=vs.110).aspx#wcf",
  "Markdown": null,
  "ApplicableApis": [
    "Not detectable via API analysis"
  ],
  "Related": null,
  "VersionBroken": "4.5",
  "VersionFixed": null,
  "IsBuildTime": false,
  "IsQuirked": false,
  "IsSourceAnalyzerAvailable": false,
  "BugLink": null,
  "Notes": null,
  "IsRetargeting": false,
  "ImpactScope": 3
}

**Parsed APIs:**
- [ ] Not detectable via API analysis